include/Metabox/Posts/Category.php

```diff
@@ -31,26 +31,38 @@ public function render() {
 				<tr>
 					<td scope="row" >
 						<?php
-						wp_dropdown_categories( array(
-							'name'         => 'smbm_mc_selected_cat',
-							'show_count'   => true,
-							'hierarchical' => true,
-							'orderby'      => 'NAME',
-							'hide_empty'   => false,
-						) );
+						$categories = get_categories( array(
+								'hide_empty' => false,
+							)
+						);
+
+						if( count($categories) > 50 ){?>
```

Comment:
We are getting all the categories. This will fail when there are huge number of categories.

Handle it based on how you did for Bulk Delete.